sdk/ai/azure-ai-inference/CHANGELOG.md

```diff
@@ -1,15 +1,24 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2024-10-03)
 
 ### Features Added
 
-### Breaking Changes
+- getModelInfo() API added for ChatCompletionsClient and EmbeddingsClient, along with sample
+- ChatRequestMessage.fromContentItems() static constructor added for image chat scenarios
+- ChatMessageImageContentItem(Path, string) constructor for image file chat scenario
+- getChoice() API added to ChatCompletions and StreamingChatMessageResponseUpdate
 
 ### Bugs Fixed
 
+- #42036: illegal char exception for newline in prompt
+- Fixed key authentication issue with non-AOAI models
```

Comment:
AOAI acronym may not be known to developers

Reply:
[Updated](https://github.com/Azure/azure-sdk-for-java/pull/42167/commits/eb885e5709a72c331c5e3cba87b2e0d59c86738f)